languageLearning/app/build.gradle

```diff
@@ -27,4 +27,6 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test:runner:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    implementation 'com.android.volley:volley:1.1.1'
```

Comment:
Already exists at line 26 of the build.gradle of android branch

Reply:
Ok. I will delete it in the next commit.